editoast/src/schema/operational_point.rs

```diff
@@ -147,3 +177,21 @@ mod test {
         from_str::<OperationalPointExtensions>(r#"{}"#).unwrap();
     }
 }
+
+#[cfg(test)]
+mod test {
+
+    use super::OperationalPoint;
+    use crate::infra::tests::test_transaction;
+
+    #[test]
+    fn test_persist() {
+        test_transaction(|conn, infra| {
+            let operational_points = (0..10)
+                .map(|_| OperationalPoint::default())
+                .collect::<Vec<OperationalPoint>>();
+
+            assert!(OperationalPoint::persist(&operational_points, infra.id, conn).is_ok());
+        });
+    }
```

Comment:
Great idea, this could be done for the other objects.

Reply:
ok will do that

Reply:
Isn't it implicitly done with the import_railjson_test_ok test in main.rs ? this more unit I guess.